CMakeLists.txt

```diff
@@ -50,13 +65,19 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     # We will make it even more strict in the future
     if(MSVC)
       target_compile_options(light_pcapng PRIVATE /W4)
-      target_compile_options(light_pcapng_static PRIVATE /W4)
     else()
       target_compile_options(light_pcapng PRIVATE -Werror)
-      target_compile_options(light_pcapng_static PRIVATE -Werror)
     endif()
 endif()
 
 if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME AND BUILD_TESTING)
     add_subdirectory(tests)
 endif()
+
+include(GNUInstallDirs)
+install(TARGETS light_pcapng)
+install(DIRECTORY include/
+    DESTINATION include
+    FILES_MATCHING
+    PATTERN *.h
+)
```

Comment:
I think it would be better to replace this with `PUBLIC_HEADER` https://stackoverflow.com/a/40285224/1438522

Reply:
ok, the next commit will use `PUBLIC_HEADER`